tools/astarte_e2e/lib/astarte_e2e/interface_provider.ex

```diff
@@ -0,0 +1,72 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2020 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteE2E.InterfaceProvider do
+  alias Astarte.Device.SimpleInterfaceProvider
+
```

Comment:
I don't like this hard coded solution, I'd rather use `priv/` directory and a JSON file, then you can load it at compile time or run time.